roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -183,10 +183,10 @@ debug mds migrator = {{ debug_mds_level }}
 rgw dns name = {{ radosgw_dns_name }}
 {% endif %}
 host = {{ hostvars[host]['ansible_hostname'] }}
-keyring = /var/lib/ceph/radosgw/ceph-rgw.{{ hostvars[host]['ansible_hostname'] }}/keyring
+keyring = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hostname'] }}/keyring
 rgw socket path = /tmp/radosgw-{{ hostvars[host]['ansible_hostname'] }}.sock
 log file = /var/log/ceph/radosgw-{{ hostvars[host]['ansible_hostname'] }}.log
```

Comment:
well we never used the cluster name for the log but this could be a bit more consistent :)